3_javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,74 @@
+// Find the individual number and the total number of Danish letters in a string.
+
+function lettersCount (danishString) {
+    const count = danishString.match(/[æøå]/g); 
+
+    if (!count) {
+        return { total: 0 }; 
+    }
+
+    const result = count.reduce((acc, vowel) => {
+        if (acc[vowel]) {
+            acc[vowel]++;
+        } else {
+            acc[vowel] = 1;
+        }
+        return acc;
+    }, {});
+
+    result.total = count.length;
+
+    return result;
+}
+
+let danishString = "Jeg har en blå bil";
+console.log(lettersCount(danishString)); // output: {å: 1, total: 1}
+
+let danishString2 = "Blå grød med røde bær";
```

Comment:
nicely done! these variables can be consts, as you don't mutate them at all :)

Reply:
Thanks for picking on this one